pagerenderer/vue/core/src/main/java/com/peregrine/pagerender/vue/models/PageModel.java

```diff
@@ -320,7 +320,13 @@ public Tag(Resource r) {
             this.path = r.getPath();
             this.path = path.substring(path.indexOf("/jcr:content"));
             this.name = r.getName();
-            this.value = r.getValueMap().get("value", String.class);
+            String tag = r.getValueMap().get("value", String.class);
+            Resource tagResource = r.getResourceResolver().getResource(tag);
+            this.value = tagResource.getValueMap().get("value", String.class);
+
+            if (this.value == null) {
+                this.value = "Please Configure";
```

Comment:
I dont think please configure is a good value for no value - that part should be left to the UI to render (a fallback to the name when rendering the tag would be better)